contracts/0.4.24/Lido.sol

```diff
@@ -240,6 +281,15 @@ contract Lido is ILido, IsContract, StETH, AragonApp {
         emit WithdrawalCredentialsSet(_withdrawalCredentials);
     }
 
+    /**
+    * @dev Sets given address as the address of LidoMevTxFeeVault contract
+    * @param _mevTxFeeVault MEV and Tx Fees Vault contract address
+    */
+    function setMevTxFeeVault(address _mevTxFeeVault) external auth(SET_MEV_TX_FEE_VAULT_ROLE) {
+        require(isContract(_mevTxFeeVault), "NOT_A_CONTRACT");
+        MEV_TX_FEE_VAULT_POSITION.setStorageAddress(_mevTxFeeVault);
```

Comment:
Could we add an event here, like in `setInsuranceFund ` for example?

Reply:
added